app/scripts/controllers/dashboard.coffee

```diff
@@ -98,12 +98,25 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
         watchCompute 'ptratioComputedMax',
           dependencies: ['allSchools']
           waitForPromise: true
-          computer: ([allSchools]) -> utils.max allSchools, 'PUPIL_TEACHER_RATIO', 10
+          computer: ([allSchools]) ->
+            max = utils.max allSchools, 'PUPIL_TEACHER_RATIO', 100
+            if max?
+              max.then (value) -> if value != $scope.range.ptratio.max
+                $scope.range.ptratio.max = value
```

Comment:
right we could watchCompute range.xxx.max base on xxxComputedMax. let me try it and it's working I will let you know. otherwise we merge it :tongue: